de.metas.business/src/main/java/org/adempiere/impexp/DiscountSchemaImportProcess.java

```diff
@@ -0,0 +1,224 @@
+package org.adempiere.impexp;
+
+import java.math.BigDecimal;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Objects;
+import java.util.Properties;
+
+import org.adempiere.ad.trx.api.ITrx;
+import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.adempiere.util.lang.IMutable;
+import org.adempiere.util.time.SystemTime;
+import org.compiere.model.I_C_BPartner;
+import org.compiere.model.I_I_DiscountSchema;
+import org.compiere.model.I_M_DiscountSchema;
+import org.compiere.model.I_M_DiscountSchemaBreak;
+import org.compiere.model.ModelValidationEngine;
+import org.compiere.model.X_I_DiscountSchema;
+import org.compiere.model.X_M_DiscountSchema;
+import org.compiere.model.X_M_DiscountSchemaBreak;
+
+import lombok.NonNull;
+
+/**
+ * Import {@link I_I_DiscountSchema} to {@link I_M_DiscountSchema}.
+ *
+ */
+public class DiscountSchemaImportProcess extends AbstractImportProcess<I_I_DiscountSchema>
+{
+	@Override
+	public Class<I_I_DiscountSchema> getImportModelClass()
+	{
+		return I_I_DiscountSchema.class;
+	}
+
+	@Override
+	public String getImportTableName()
+	{
+		return I_I_DiscountSchema.Table_Name;
+	}
+
+	@Override
+	protected String getTargetTableName()
+	{
+		return I_M_DiscountSchema.Table_Name;
+	}
+
+	@Override
+	protected void updateAndValidateImportRecords()
+	{
+		MDiscountSchemaImportTableSqlUpdater.updateDiscountSchemaImportTable(getWhereClause());
+	}
+
+	@Override
+	protected String getImportOrderBySql()
+	{
+		return I_I_DiscountSchema.COLUMNNAME_ProductValue;
```

Comment:
as I understand, we are updating the discount schemas, one for each bpartner.
why here is are sorting by ProductValue and why not by C_BPartner_ID.